src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -2,6 +2,39 @@
 
     Handler for time interval CLI arguments
 
+    One Argument
+    ------------
+
+    Date:
+        Gives the range for that day.
+        "-t 2019-04-01" (start: 2019-04-01 00:00:00, end: 2019-04-01 23:59:59 )
+
+    TimeStamp
+        Gives a range from the provided timestamp plus one day.
+        "-t 1554132392" (start: 1554132392, end: 1554218791)
+
+    Timerange
+        Gives X amount in the past NOT including the current second.
+        "-t 1h" with now:1000 (start:940, end:999)
```

Comment:
The '1h" was a mistake, it should say `1m`.
-t 3600s is still 3600 seconds worth of data, because it is intended that the ranges' end is to be used in the query eg.
```d
for ( long i = range.begin; i <= range.end; ++i )
```

`-t 1s` would mean you get the previous second of data. So you'd get the data from just 999.

-t 0h i believe should be an error (if it's not already)

Reply:
Added a commit that treats 0 as an error.